src/chrome/content/rules/Federal_Trade_Commission.xml

```diff
@@ -43,14 +35,11 @@ Fetch error: http://new.ftc.gov/ => https://new.ftc.gov/: (6, 'Could not resolve
 	<target host="consumidor.ftc.gov" />
 	<target host="www.consumidor.ftc.gov" />
 	<target host="edit.ftc.gov" />
-	<target host="new.edit.ftc.gov" />
-	<target host="eis.ftc.gov" />
```

Comment:
This host serves an incomplete certificate chain. Please add a comment about this issue.